### PR TITLE
First version

### PR DESCRIPTION
**Pull request by trocolit2**
Please check my code!

### CODE REVIEW

File: src/ImageEnhanceViaFusion.cpp [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+#include <vector>
+#include <iostream>
+
+
+#include <opencv2/imgproc/imgproc.hpp>
+
+namespace underwater_image_tools {
+
+
+// percent should have values between 0.0001 and 0.05
```
Comment (romulogcerqueira) [inline src/ImageEnhanceViaFusion.cpp]: This method should contain a check for `percent_channel` value

File: src/ImageEnhanceViaFusion.cpp [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+cv::Mat colorBalance( cv::Mat image, const cv::Scalar& percent_channel ){
```
Comment (romulogcerqueira) [inline src/ImageEnhanceViaFusion.cpp]: Once you are passing, modifying and returning the same `image`, you can pass this `cv::Mat` as reference and avoid copies.

File: test/ImageEnhanceViaFusion_test.cpp [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+
+
+#include <underwater_image_tools/ImageEnhanceViaFusion.hpp>
+
+
+#define PATH_DATASET "../test/dataset/"
```
Comment (romulogcerqueira) [inline test/ImageEnhanceViaFusion_test.cpp]: Try to use this as absolute path, for example:

<pre>
std::string current_path(__FILE__);
current_path = current_path.substr(0, current_path.find_last_of("/"));
</pre>

**Comment by trocolit2** [comment]
Code changed. Please review!

**Comment by romulogcerqueira** [comment]
It's good to me. Merging here.

**Pull request merged**